index.html

```diff
@@ -47,12 +47,11 @@ <h3>runit</h3>
 			<p>runit is a simple and effective approach to initialize the system with reliable service supervision. Refer to the <a href="https://docs.voidlinux.org/config/services/index.html">Void Handbook</a> for an introduction.</p>
 		</div>
 	</div>
-        <div class="row">
-                <div class="col-md-4">
-                        <h3>LibreSSL</h3>
-                        <p>We were the first distribution to switch to <em>LibreSSL</em> by default, replacing <em>OpenSSL</em>.</p>
-			<p>Due to the <a href="https://en.wikipedia.org/wiki/Heartbleed">Heartbleed</a> fiasco, we believe that the <a href="https://www.openbsd.org">OpenBSD</a> project has qualified and <em>pro-active</em> developers to provide a more secure alternative.</p>
-		</div>
+		<div class="row">
+			<div class="col-md-4">
+				<h3>Musl and glibc</h3>
+				<p>Void Linux supports both the <a href="http://musl.libc.org/">Musl</a> and <a href="https://www.gnu.org/software/libc/">glibc</a> libcs, patching incompatible projects when necessary and working with upstream projects to make their code bases more standards compliant and compatible.
```

Comment:
"libcs" is a strange construct. I recommend "C libraries".

Reply:
Perhaps "C standard libraries"?

Reply:
Should be fixed now.

Reply:
Looks good to me.